packages/formation/js/additional-info.js

```diff
@@ -0,0 +1,66 @@
+function uniqueId() {
+  return Math.floor(Math.random() * Math.floor(100000));
+}
+
+function qSA(rootNode, selector) {
```

Comment:
not sure what i want to do with these- they're in a separate file in `vets-website` but i don't think anything else uses them. open to ideas here.